[developer] Compile error for Object.values

**Describe the bug**
I was trying to compile lexical-models in Ubuntu 18.04.2  LTS and got the following compile error
```
Building model /home/darcy/src/lexical-models/release/nrc/en.mtnt/
../../../../../keymanapp/developer/js/lexical-model-compiler/build-trie.ts:347:21 - 
error TS2339: Property 'values' does not exist on type 'ObjectConstructor'.

347       return Object.values(node.children)
                        ~~~~~~
Found 1 error.

Compile phase 1 failed for /home/darcy/src/lexical-models/release/nrc/en.mtnt/
Aborting with error 2
```
(My local clone of keymanapp/keyman is `keymanapp`)

From [SO](https://stackoverflow.com/questions/42966362/how-to-use-object-values-with-typescript/42967397), `values` is ES7 while we're using ES6.

Should we try to use `Object.keys` instead of `Object.values`?

**Expected behavior**

The lexical-models repo should successfully compile

-----

### Configuration ###
OS: Ubuntu 18.04.02
nodejs: 8.16.0
npm: 6.4.1

--- a/developer/js/lexical-model-compiler/build-trie.ts
+++ b/developer/js/lexical-model-compiler/build-trie.ts
@@ -344,8 +344,8 @@ namespace Trie {
         .map(entry => entry.weight)
         .reduce((acc, count) => acc + count, 0);
     } else {
-      return Object.values(node.children)
-        .map((child) => sumWeights(child))
+      return Object.keys(node.children)
+        .map((key) => sumWeights(node.children[key]))
         .reduce((acc, count) => acc + count, 0);
     }
   }

--- a/developer/js/tests/test-compile-trie.ts
+++ b/developer/js/tests/test-compile-trie.ts
@@ -21,9 +21,8 @@ describe('LexicalModelCompiler', function () {
       }, SyntaxError);
       // TODO: Mock LMLayerWorker.loadModel()
   
-      console.log(code);
-      // Sanity check: the word list has three total
-      // unweighted words, with a total weight of 3!
+      // Sanity check: the word list has three total unweighted words, with a
+      // total weight of 3!
       assert.match(code, /\btotalWeight\b["']?:\s*3\b/);
     });
   })

--- a/developer/js/tests/test-compile-trie.ts
+++ b/developer/js/tests/test-compile-trie.ts
@@ -20,6 +20,11 @@ describe('LexicalModelCompiler', function () {
         eval(code);
       }, SyntaxError);
       // TODO: Mock LMLayerWorker.loadModel()
+  
+      console.log(code);
+      // Sanity check: the word list has three total
+      // unweighted words, with a total weight of 3!
+      assert.match(code, /\btotalWeight\b["']?:\s*3\b/);
     });
   })
 });

